dash_player/__init__.py

```diff
@@ -10,37 +10,33 @@
 from ._imports_ import *
 from ._imports_ import __all__
 
-if not hasattr(_dash, 'development'):
-    print('Dash was not successfully imported. '
-          'Make sure you don\'t have a file '
-          'named \n"dash.py" in your current directory.', file=_sys.stderr)
+if not hasattr(_dash, "development"):
+    print(
+        "Dash was not successfully imported. "
+        "Make sure you don't have a file "
+        'named \n"dash.py" in your current directory.',
+        file=_sys.stderr,
+    )
     _sys.exit(1)
 
 _basepath = _os.path.dirname(__file__)
-_filepath = _os.path.abspath(_os.path.join(_basepath, 'package.json'))
+_filepath = _os.path.abspath(_os.path.join(_basepath, "package.json"))
 with open(_filepath) as f:
     package = json.load(f)
 
-package_name = package['name'].replace(' ', '_').replace('-', '_')
-__version__ = package['version']
+package_name = package["name"].replace(" ", "_").replace("-", "_")
+__version__ = package["version"]
 
 _current_path = _os.path.dirname(_os.path.abspath(__file__))
 
 _this_module = _sys.modules[__name__]
 
 
-_js_dist = [
-    {
-        'relative_package_path': 'dash_player.min.js',
-        'dev_package_path': 'dash_player.dev.js',
-        
-        'namespace': package_name
-    }
-]
+_js_dist = [{"relative_package_path": "dash_player.min.js", "namespace": package_name}]
```

Comment:
Put the `.js.map` file here too. Make sure it has `dynamic: True` so it's only loaded when the browser requests it.

Also assuming we're going to publish this to NPM we can include `external_url`. The end state should look kind of like [dash.html](https://github.com/plotly/dash/blob/dev/components/dash-html-components/dash_html_components_base/__init__.py#L34-L52)

As discussed on slack, `dash-player` is already taken on NPM, and it looks like that means we also [can't use `dash_player`](https://blog.npmjs.org/post/168978377570/new-package-moniker-rules.html)

So let's namespace it - name the package `@plotly/dash-player`

See if you can figure out how that works with `unpkg` - can you just put a namespaced package name in the URL? You can test that with an existing namespaced package like https://www.npmjs.com/package/@plotly/dash-component-plugins

Reply:
Tested and changed!